docker-compose.yml

```diff
@@ -467,6 +467,26 @@ services:
     ports:
       - "15000:15000"
 
+  #
+  # The second version of "live replay" server
+  #
+  faf-aio-replayserver:
+    container_name: faf-aio-replayserver
+    build: git://github.com/FAForever/faf-aio-replayserver
```

Comment:
This one should include a tag name as well:

`git://github.com/FAForever/faf-aio-replayserver#0.1`